tests/integration/README.md

```diff
@@ -79,68 +80,52 @@ echo 'monitord.rotate_log=0' >> $wazuh_path/etc/local_internal_options.conf
 choco install jq
 ```
 
-- Install Python dependencies
+- Install integration tests dependencies
 
 ```shell script
-pip install pytest freezegun jsonschema pyyaml==5.4 psutil paramiko distro pywin32 pypiwin32 wmi pandas==0.25.3 pytest-html==2.0.1 numpydoc==0.9.2
+cd wazuh-qa
+python -m pip install -r requirements.txt
 ```
 
-- Change `time-reconnect` from `C:\Program Files (x86)\ossec-agent\ossec.conf`
-
-```xml
-<time-reconnect>99999999999</time-reconnect>
-```
-
-- Add some internal options
+- Install wazuh-testing framework
 
 ```shell script
-# Enable debug 2
-echo 'syscheck.debug=2' >> "C:\Program Files (x86)\ossec-agent\local_internal_options.conf"
-
-# Disable log rotation
-echo 'monitord.rotate_log=0' >> "C:\Program Files (x86)\ossec-agent\local_internal_options.conf"
+cd deps/wazuh_testing/
+python setup.py install
 ```
 
-- Restart **Wazuh** using the GUI
+> [!NOTE]
+> When developing tests, utilize the following command to seamlessly update the framework with each modification.
+```
+python -m pip install . -e
+```
 
 ### MacOS
 
-- Install **Wazuh**
+- Install and start **Wazuh**
 
-- Install Python and its dependencies
+> [!NOTE]
+> Remember to register the agent to the manager
 
-```shell script
 # Install Python
-brew install python3
 
-# Install dependencies
-brew install autoconf automake libtool
-
-# Install Python libraries
-pip3 install filetype freezegun jq jsonschema lockfile numpydoc psutil pytest-html pytest-testinfra pyyaml
+```shell script
+brew install python3
 ```
 
-- Add some internal options and restart
```

Comment:
More internal options to take into consideration.

Reply:
Done 15215b391ba5d18eae8923b418137fd9f649b541